### DISCUSSION TITLE
About web API issues

### DISCUSSION THREAD

**Discussion by 164ebr** [plan-player-analytics/Plan#3469] [Q&A]
Does the plugin provide any web API for external programs to access? For example, a GET request

**Comment by 164ebr** [comment]
> Does the plugin provide any web API for external programs to access? For example, a GET request

ok i get , "http://127.0.0.1:8804/v1/network/overview"
Here is the data I need

**Reply by AuroraLS3** [reply]
Swagger docs offer some documentation though some are missing due to missing annotations
